### ISSUE TITLE
STARNet: Sensor Trustworthiness and Anomaly Recognition via Approximated Likelihood Regret for Robust Edge Autonomy. (arXiv:2309.11006v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#27776]
STARNet: Sensor Trustworthiness and Anomaly Recognition via Approximated Likelihood Regret for Robust Edge Autonomy. (arXiv:2309.11006v1 [cs.RO])<br>
https://ift.tt/dY1p9Z5<br>
<p>Complex sensors such as LiDAR, RADAR, and event cameras have proliferated in
autonomous robotics to enhance perception and understanding of the environment.
Meanwhile, these sensors are also vulnerable to diverse failure mechanisms that
can intricately interact with their operation environment. In parallel, the
limited availability of training data on complex sensors also affects the
reliability of their deep learning-based prediction flow, where their
prediction models can fail to generalize to environments not adequately
captured in the training set. To address these reliability concerns, this paper
introduces STARNet, a Sensor Trustworthiness and Anomaly Recognition Network
designed to detect untrustworthy sensor streams that may arise from sensor
malfunctions and/or challenging environments. We specifically benchmark STARNet
on LiDAR and camera data. STARNet employs the concept of approximated
likelihood regret, a gradient-free framework tailored for low-complexity
hardware, especially those with only fixed-point precision capabilities.
Through extensive simulations, we demonstrate the efficacy of STARNet in
detecting untrustworthy sensor streams in unimodal and multimodal settings. In
particular, the network shows superior performance in addressing internal
sensor failures, such as cross-sensor interference and crosstalk. In diverse
test scenarios involving adverse weather and sensor malfunctions, we show that
STARNet enhances prediction accuracy by approximately 10% by filtering out
untrustworthy sensor streams. STARNet is publicly available at
\url{https://ift.tt/kQAzPsn}.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
